definitions/procedures/pulpcore/rpm_datarepair.rb

```diff
@@ -0,0 +1,22 @@
+module Procedures::Pulpcore
+  class RpmDatarepair < ForemanMaintain::Procedure
```

Comment:
RpmDataRepair would look better,  but the Pulp command does call it datarepair all lowercase...